Modelica/Units.mo

```diff
@@ -781,6 +781,8 @@ end UsersGuide;
         min=0);
     type StoichiometricNumber = Real (final quantity="StoichiometricNumber",
           final unit="1");
+    type StoichiometricCoefficient = Real (final quantity="StoichiometricCoefficient",
```

Comment:
```suggestion
    type StoichiometricCoefficient = Real (final quantity="StoichiometricCoefficient",min=0,
```
(Well, normally they are `>0` but we cannot write that - and in case someone has a general formula it doesn't hurt to allow 0.)

Reply:
The reason wasn't just to be pedantic with the min-value but also to have as a differentiator and reminder of which one is which.

Reply:
Sure.  It's a good idea.